platform-core-module/src/main/java/com/softicar/platform/core/module/user/AGUser.java

```diff
@@ -247,4 +252,31 @@ public IDisplayString toDisplay() {
 
 		return toDisplayWithoutId();
 	}
+
+	public UserPreferences getPreferences() {
+
+		UserPreferences preferences = null;
+		try {
+			preferences = new Gson().fromJson(getPreferencesJson(), UserPreferences.class);
+		} catch (Exception exception) {
+			DevNull.swallow(exception);
+			Log.ferror("Failed to retrieve preferences for user '%s'. Using defaults.", toDisplay());
+		}
```

Comment:
could be extracted to loadPreferences or something

Reply:
ok